src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseAssignedUserRolesController.java

```diff
@@ -135,7 +133,6 @@ public CaseAssignedUserRolesController(ApplicationParams applicationParams,
         return ResponseEntity.status(HttpStatus.CREATED).body(new CaseAssignedUserRolesResponse(ADD_SUCCESS_MESSAGE));
     }
 
-    @Transactional
     @DeleteMapping(
```

Comment:
better to have transactional on delete methods, not on controller thought

Reply:
The implementation is CaseAccessOperation. removeCaseUserRoles is @Transactional